app/controllers/comment_controller.php

```diff
@@ -14,8 +15,15 @@ public function view()
         $comments = $comment->getAll($pagination->start_index-1, $pagination->count+1, $thread->id);
         $pagination->checkLastPage($comments);
 
-        foreach ($comments as $comment) {
-            $comment->user_id = User::getUsernameById($comment->user_id);
+        foreach ($comments as &$comment) {
+            $comment->username = User::getUsernameById($comment->user_id);
+            $comment->likecount = Likes::countLike($comment->id);
+        }
+
+        if($sort === 'comment') {
```

Comment:
Done po, I included po the table name as magic strings din po. :smile:  :bow:  https://github.com/renzosunico/MyClassroom/commit/af616b05b2ae0f497e6ee8913c847f3fb0ba753b https://github.com/renzosunico/MyClassroom/commit/1bdf42eed7d4a82b34f590aec10a9c7d32cb61e9